src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -1434,18 +1476,23 @@ public void LoadKeysLookupLock()
             {
                 foreach (Wallet wallet in this.Wallets)
                 {
-                    IEnumerable<HdAddress> addresses = wallet.GetAllAddresses(a => true);
-                    foreach (HdAddress address in addresses)
+                    foreach (HdAccount account in wallet.GetAccounts(a => true))
```

Comment:
Why the split here? `GetAllAddresses` isn't working as expected?

Reply:
@zeptin claim this was a bug before, I think he is right, thoughts?

Reply:
The possible bug I was referring to is in `RemoveTransactionsByIdsLocked`; `a => true` was getting properly passed in `LoadKeysLookupLock` already, but it is missing there.